src/earl-loader.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+const globby = require('globby')
+
+/**
+ * Load jsonld/earl report(s)
+ * - if there are multiple reports, this function concatanates them into one report
+ *
+ * @method earlLoader
+ * @param {String|String[]} pathOrGlob file path to jsonld/earl report
+ * @returns {Object[]}
+ */
+const earlLoader = async pathOrGlob => {
```

Comment:
Why is this async? There's nothing async in this method. I do agree this should be async, since it does file IO. Best to just use async methods if you can.